github.go

```diff
@@ -23,6 +23,7 @@ type Repositories interface {
 
 type Issues interface {
 	AddLabelsToIssue(owner, repo string, number int, labels []string) ([]github.Label, *github.Response, error)
+	RemoveLabelForIssue(owner, repo string, number int, label string) (*github.Response, error)
```

Comment:
Can't rename the methods in these interfaces, as I'm the types coming from go-github package into these interfaces. This means that the signature here must exactly match that [defined here](https://godoc.org/github.com/google/go-github/github#IssuesService.RemoveLabelForIssue) (the argument names, can still differ, but the type signature must be the same)